api/proto/banyandb/property/v1/rpc.proto

```diff
@@ -56,7 +56,7 @@ message GetResponse {
 }
 
 message ListRequest {
-    banyandb.common.v1.Metadata contaner = 1;
+    banyandb.common.v1.Metadata container = 1;
```

Comment:
When you change this file, the references in the source codes also have to be changed.

Reply:
I see, and I wonder is "make generate" at root directory all i need to do ? By the way, if i run "make generate", there are over a hundred files that have their format changed, most of which are removed with some whitespace lines in the "import" module. Is that ok?

Reply:
> I see, and I wonder is "make generate" at root directory all i need to do ? By the way, if i run "make generate", there are over a hundred files that have their format changed, most of which are removed with some whitespace lines in the "import" module. Is that ok?

A hundred files? Not possible, it will only make necessary changes. I guess in your case, only `rpc.pb.go` will change.

Reply:
![1654164745(1)](https://user-images.githubusercontent.com/49940663/171608063-b4057261-5056-4d15-8fea-6689e67e5fc1.png)
emmm, this is the result of "make generate", it may adjust the format ?

Reply:
The formatted codes (in 3 blocks) are generated by `goimports`. Have you installed the latest `goimports` globally?

Reply:
To issue `make format` after generating codes.

Reply:
> The formatted codes (in 3 blocks) are generated by `goimports`. Have you installed the latest `goimports` globally and make sure it can be found within the `PATH`?
> 
> Pls refer to the official document, https://pkg.go.dev/golang.org/x/tools/cmd/goimports

I ignored goimports error information. My fault, thanks!!